### DISCUSSION TITLE
Add more permissions to /cmi armorstand templates option

### DISCUSSION THREAD

**Discussion by MrCrazys** [Zrips/CMI#6915] [Enhancement]
Hello,

It would be really useful if more permissions could be added to the templates section of armour stands. Right now, there is only one permission that allows access to everything template related, also meaning that players can delete other player's templates, and more. Adding permssions such as `cmi.command.armorstand.template.delete.own`, `cmi.armorstand.template.list`, etc.

Thanks,

Kind regards,

MrCrazys

**Comment by mrfloris** [comment]
Yes, the new template system is for the admin only. so they can create and save armorstands, and get them as an item, which they can then give to players. It's not a player feature.
